src/Controller/BlogController.php

```diff
@@ -61,6 +61,7 @@ public function tag(string $tag, int $page = 1, int $perPage = 20): Response
     public function rss(): Response
     {
         $articles = $this->manager->getContents(Article::class, ['date' => false]);
+        $articles = array_filter($articles, fn (Article $article) => $article->date > new \DateTime('-6 months'));
```

Comment:
```suggestion
        $articles = $this->manager->getContents(Article::class, ['date' => false], '_.date > date("-6 months")');
```

🤓 

https://stenopephp.github.io/Stenope/loading-content/#an-expressionlanguage-expression

```bash
symfony console debug:stenope:content article --filter="_.date > date('-6 months')" --order="-date"
```